lib/tasks/package.js

```diff
@@ -25,6 +26,8 @@ class PackageTask extends Task {
     } else {
       // Assemble the project and pass to forge
       projectPath = quickTemp.makeOrRemake(this, '-tmpPath');
+      childProcess.execSync(`chmod a+x ${projectPath}`);
```

Comment:
Looping back, this is reasonably out of scope due to `Builder` needing enhanced permissions:

```